script.js

```diff
@@ -0,0 +1,14 @@
+// Select elements
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.navlink');
+// const span = document.getElementsByClassName('bar')[2];
+// Add Event Listener
```

Comment:
The mobile menu works as expected. However, kindly remove unused pieces of code in order to make the code more concise.